openedx/core/lib/xblock_serializer/block_serializer.py

```diff
@@ -146,6 +149,9 @@ def _serialize_html_block(self, block) -> etree.Element:
         """
         olx_node = etree.Element("html")
         olx_node.attrib["url_name"] = block.scope_ids.usage_id.block_id
+        if self.add_upstream:
+            olx_node.attrib["upstream"] = str(block.scope_ids.usage_id)
+            olx_node.attrib["upstream_version"] = str(block.scope_ids.version)
```

Comment:
Is this required only for html blocks? I can see that it is only used for containers and it fails here when copying a unit that contains an html block